src/diff.c

```diff
@@ -1863,6 +1872,16 @@ int git_diff_commit_as_email(
 	opts.summary = git_commit_summary(commit);
 	opts.author = git_commit_author(commit);
 
+	/* The first line of the commit message is identical to
+	 * the summary and thus needs to be skipped */
+	if ((ptr = git_commit_message(commit)) != NULL) {
+		/* Find end of first line */
+		ptr = strpbrk(ptr, "\r\n");
```

Comment:
Well, actually I'm not sure about this. Just searched git-am(1), git-format-patch(1) and git-apply(1), but there was no mention of line endings. There are some platforms that use `\r` as line endings, that's why I used this.

I think it might make sense to expose a new function `git_commit_body`, that will return a pointer to the commit message excluding the summary. This could be used here, then, and might prove useful for others.